components/logins/android/build.gradle

```diff
@@ -68,8 +80,11 @@ dependencies {
     // avoiding other configurations from being resolved.  Tricky!
     testImplementation files(configurations.jnaForTest.copyRecursive().files)
     testImplementation 'junit:junit:4.12'
-    testImplementation 'org.robolectric:robolectric:3.8'
+    testImplementation 'org.robolectric:robolectric:4.2.1'
     testImplementation 'org.mockito:mockito-core:2.21.0'
+    testImplementation 'androidx.test:core-ktx:1.2.0'
+    testImplementation 'androidx.work:work-testing:2.2.0'
+    testImplementation "org.mozilla.telemetry:glean-forUnitTests:$glean_version"
```

Comment:
@Dexterp37 oh, this was the other thing that was a bit weird about setting up the tests - having to load the `-forUnitTests` version of the direct glean dependency for tests, while consuming the implementation version of glean through a-c. I only knew to look for this because we have our own `-forUnitTest` shenanigans in this repo.

I wonder if there's a way to expose this through a-c rather than depending directly on glean?

Reply:
That's a great point. We're now trying to to this in the plugin, so this will hopefully go away with mozilla-mobile/android-components#4985. Initially, we'll just be exposing the version so that consumers don't need to "keep versions in sync". We'll also try to see if we can manually inject the dependency in test configurations.